packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.generated.ts

```diff
@@ -112,26 +112,26 @@ use_old_EnumDeclaration_CompressionAlgorithms(
 /*
 * Validate forward compat by using old type in place of current type
 * If breaking change required, add in package.json under typeValidation.broken:
-* "EnumDeclaration_ContainerMessageType": {"forwardCompat": false}
+* "RemovedEnumDeclaration_ContainerMessageType": {"forwardCompat": false}
```

Comment:
This I would deprecate and leave in `index` for now.  We will have to deal with the strikethrough in our own codebase for a few months.